src/views/GovernanceTaskList/index.tsx

```diff
@@ -1,377 +1,217 @@
-import React, { useEffect } from 'react';
-import { useTranslation } from 'react-i18next';
-import { useDispatch } from 'react-redux';
-import { Link, useHistory, useParams } from 'react-router-dom';
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
 import { useQuery } from '@apollo/client';
-import {
-  Breadcrumb,
-  BreadcrumbBar,
-  BreadcrumbLink
-} from '@trussworks/react-uswds';
-import { DateTime } from 'luxon';
+import { Alert, Button, Grid, GridContainer } from '@trussworks/react-uswds';
```

Comment:
On second thought, is there any issue with the current regular Alert and how it matches the figma?

It looks like the shared Alert can be dismissible, which I don't think is a requirement here?

If the regular is serving the requirements then I'd like to leave it. Shared is wrapping the truss alert anyway.

Reply:
It looks like the shared alert is only closable (by default) on `success` or `error`, and since this one's `<Alert type="info"` I think it will behave pretty similarly (but not sure if there's any other reasons we want to use the shared one)

Reply:
> if we want to just use the shared one everywhere, might make sweeping updates easier to make?

Yeah this was basically the only reason I pointed it out. When Patrick made the Alert closable, he updated all the Alerts to use the shared component. I think mixing usage of both Truss's components and our own for the same element just adds another step if/when we need to make updates, so it would be nice to just default to one type if possible.